manubot/process/tests/manuscripts/example/content/metadata.yaml

```diff
@@ -2,6 +2,7 @@
 title: "Example manuscript for testing"
 keywords:
   - manubot
+date: 2022-11-22
```

Comment:
Running the following tests:

```
pytest manubot/process/tests/test_process_command.py
```

Now produces the following output variables in `manubot/process/tests/manuscripts/example/output/variables.json`:

```json
  "manubot": {
    "date": "2022-11-22",
    "date_long": "November 22, 2022",
    "generated": "2022-11-12T08:50:02-05:00",
    "generated_date_long": "November 12, 2022",
```